prql-compiler/src/semantic/lowering.rs

```diff
@@ -558,8 +559,40 @@ impl Lowerer {
                     rq::ExprKind::SString(vec![InterpolateItem::String(ident.name)])
                 }
             }
-            pl::ExprKind::All { .. } => {
-                bail!(Error::new_simple("Wildcards cannot be used here").with_span(ast.span))
+            pl::ExprKind::All { except, .. } => {
+                let mut targets = Vec::new();
+
+                for target_id in &ast.target_ids {
+                    match self.node_mapping.get(target_id) {
+                        Some(LoweredTarget::Compute(cid)) => targets.push(*cid),
+                        Some(LoweredTarget::Input(input_columns)) => {
+                            targets.extend(input_columns.values().map(|(c, _)| c))
+                        }
+                        _ => {}
+                    }
+                }
+
+                // this is terrible code
```

Comment:
lol, actually underrated comment — we should have more of these! It's also a nice way for someone to make a very small rust contribution!